src/ref.rs

```diff
@@ -643,7 +800,10 @@ where
     /// there is no conflict with a method on the inner type.
     #[must_use = "has no side effects"]
     #[inline(always)]
-    pub fn into_mut(r: Self) -> &'a mut T {
+    pub fn into_mut(r: Self) -> &'a mut T
+    where
+        T: FromBytes,
+    {
```

Comment:
Why not just create a new impl block? Mixing bounds on the impl block and bounds on the function feels odd and error-prone (esp with future changes).

Reply:
When I read bound-heavy documentation (e.g., nalgebra), I've always been grateful when methods with related bounds are grouped together — so much so, I [wrote a blog post about it](https://jack.wrenn.fyi/blog/rustdocing-nalgebra/).

Reply:
Ack